server-api/src/main/java/org/sonarsource/sonarlint/core/serverapi/hotspot/HotspotApi.java

```diff
@@ -192,10 +204,9 @@ private static HotspotReviewStatus getStatus(Hotspots.SearchWsResponse.Hotspot h
     return HotspotReviewStatus.TO_REVIEW;
   }
 
-  private static String getShowUrl(String hotspotKey, String projectKey) {
+  private static String getShowUrl(String hotspotKey) {
     return HOTSPOTS_SHOW_API_URL
-      + "?projectKey=" + urlEncode(projectKey)
-      + "&hotspot=" + urlEncode(hotspotKey);
+      + "?hotspot=" + urlEncode(hotspotKey);
```

Comment:
Why were we using `projectKey` as a parameter? API documentation doesn't show anything for this param

Reply:
I have no idea.. I thought it was required by some old versions of SonarQube but I was not able to find it